tools/ci/stage.sh

```diff
@@ -21,8 +21,8 @@ STAGE_COMPILE="compile"
 STAGE_CORE="core"
 STAGE_PYTHON="python"
 STAGE_TABLE="table"
-STAGE_CONNECTORS="connectors"
-STAGE_KAFKA_GELLY="kafka/gelly"
+STAGE_CONNECTORS_1="connectors_1"
+STAGE_CONNECTORS_2="connectors_2"
```

Comment:
I don't like how these two stages have a common prefix. In AzureCI, the end of the name is sometimes cut of leaving both stages with the same label + "..." as a prefix. Can't we come up with something more descriptive?

Reply:
any suggestions? there isn't a particular pattern (apart from formats/reporters/filesystems being in connector_1), and I'm not keen on a naming scheme that includes specific modules (e.g., "kafka/pulsar") as I'm not keen on having to rename stuff when we externalize more stuff.

Ultimately the lifetime of these stages is limited anyway; until the connectors have been externalized,

Reply:
what about something like `stage_[1|2]_connectors` or `set_[1|2]_connectors` or just `[1|2]_connectors`?

Reply:
would connect_1 be short enough?

Reply:
I mean, in the end it depends on the screen and font size of the user. One could also argue that `test_ci` is already problematic as a prefix for the setup I had in mind. Considering that (and agreeing that the index being at the end of the name is more naturally) I'd be ok with just keeping it like that for now. Probably, it's just overengineering. If it's really an issue we could address it later on...